packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -550,87 +602,101 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
   const variables: CSSOutput['variables'] = [];
 
   // Quasis are the string pieces of the template literal - the parts around the interpolations
-  const literalResult = node.quasis.reduce<string>((acc, quasi, index): string => {
-    const nodeExpression = node.expressions[index] as t.Expression | undefined;
-
-    if (
-      !nodeExpression ||
-      (t.isArrowFunctionExpression(nodeExpression) && t.isLogicalExpression(nodeExpression.body))
-    ) {
-      const suffix = meta.context === 'keyframes' ? '' : ';';
-      return acc + quasi.value.raw + suffix;
-    }
+  const literalResult = node.quasis.reduce<string>(
+    (acc: string, quasi: t.TemplateLitteral, index: number): string => {
+      const nodeExpression = node.expressions[index] as t.Expression | undefined;
+
+      // Deal with any negative values such as:
+      // margin: -${gridSize}, top: -${gridSize} etc.
+      if (quasi.value.raw.endsWith('-') && !t.isArrowFunctionExpression(nodeExpression)) {
+        quasi.value.raw = quasi.value.raw.substring(0, quasi.value.raw.length - 1);
+        quasi.value.cooked = quasi.value.cooked.substring(0, quasi.value.cooked.length - 1);
+        convertNegativeCssValuesToUnaryExpression(nodeExpression);
```

Comment:
Any reason why 
```
quasi.value.raw = quasi.value.raw.substring(0, quasi.value.raw.length - 1);
quasi.value.cooked = quasi.value.cooked.substring(0, quasi.value.cooked.length - 1);
```
are not placed inside  `convertNegativeCssValuesToUnaryExpression`?

Reply:
I originally was passing the `quasi` in - now i'm returning the new `nodeExpression` from the `convertNegativeCssValuesToUnaryExpression` function


```
if (quasi.value.raw.endsWith('-') && !t.isArrowFunctionExpression(nodeExpression)) {
        quasi.value.raw = trimUnaryOperator(quasi.value.raw);
        quasi.value.cooked = trimUnaryOperator(quasi.value.cooked);
        nodeExpression = convertNegativeCssValuesToUnaryExpression(nodeExpression);
      }
```      

> Based on the comment @param quasi quasi information about the current node

I've gone ahead and removed this param